bcmp/neighbors.c

```diff
@@ -275,3 +406,52 @@ void bcmp_neighbor_invoke_discovery_cb(bool discovered,
     NEIGHBOR_DISCOVERY_CB(discovered, neighbor);
   }
 }
+
+/*!
+  @brief Send neighbor table request to node(s)
+
+  @param target_node_id - target node id to send request to (0 for all nodes)
+  @param *addr - ip address to send to send request to
+  @ret ERR_OK if successful
+*/
+BmErr bcmp_request_neighbor_table(uint64_t target_node_id, const void *addr,
+                                  NeighborRequestCallback request,
+                                  BmTimerCallback timeout) {
+  BmErr err = BmOK;
+  BcmpNeighborTableRequest neighbor_table_req = {.target_node_id =
+                                                     target_node_id};
+  TARGET_NODE_ID = target_node_id;
+  if (NEIGHBOR_TIMER) {
+    bm_timer_delete(NEIGHBOR_TIMER, 10);
+  }
+  NEIGHBOR_TIMER =
+      bm_timer_create("topology_timer", bcmp_neighbor_timer_timeout_s * 1000,
```

Comment:
nit: I don't think we need the timer name anymore, but if we want to keep it maybe naming it to `neighbor_request_timer` or something would be nice.

Reply:
Not a bad nit I agree with that statement